<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera</class>
 <widget class="QWidget" name="Camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="camera_image_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>9</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="cameraImageContainer"/>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Acquisition</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Device name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="device_label">
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Camera type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="camera_type_label">
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Camera model</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="camera_model_label">
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>State</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="TaurusLabel" name="state_tlabel"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="TaurusLabel" name="status_tlabel"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Acq. Status</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="TaurusLabel" name="acq_status_label"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Acquisition</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="start_acquisition_button">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_acquisition_button">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Frame number</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="TaurusLabel" name="acq_framenumber_label">
          <property name="model">
           <string/>
          </property>
          <property name="useParentModel">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Exposure time (ms)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="MAXLineEdit" name="acq_expo_time">
          <property name="inputMask">
           <string/>
          </property>
          <property name="model">
           <string>acq_expo_time</string>
          </property>
          <property name="useParentModel">
           <bool>false</bool>
          </property>
          <property name="autoApply">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Gain</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="MAXLineEdit" name="gain_label"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Trigger mode</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="TaurusValueComboBox" name="trigger_mode_combobox">
          <property name="autoApply">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Image width</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="TaurusLabel" name="image_width_label"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Image height</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="TaurusLabel" name="image_height_label"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Rotation</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="TaurusValueComboBox" name="image_rotation_combobox">
          <property name="autoApply">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Binning</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="image_bin_spinbox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="image_save_button">
          <property name="text">
           <string>Save Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>BPM</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>ROI</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TaurusLabel" name="roi_label">
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="model">
             <string>ROI</string>
            </property>
            <property name="useParentModel">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Auto ROI</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TaurusValueCheckBox" name="auto_roi_checkbox">
            <property name="text">
             <string/>
            </property>
            <property name="showText">
             <bool>false</bool>
            </property>
            <property name="autoApply">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="bpm_show_position_checkbox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Show beam position</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Beam parameters</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="3">
              <widget class="TaurusLabel" name="beam_fwhm_x_label">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Center Y:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Intensity:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="3">
              <widget class="TaurusLabel" name="beam_fwhm_y_label">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="TaurusLabel" name="beam_center_x_label">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Center X:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>FWHM Y:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="TaurusLabel" name="beam_center_y_label">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="TaurusLabel" name="beam_intensity_label">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>FWHM X:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="4">
              <layout class="QVBoxLayout" name="bpm_profile_x_layout"/>
             </item>
             <item row="3" column="0" colspan="4">
              <layout class="QVBoxLayout" name="bpm_profile_y_layout"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="bpm_layout"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_5">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="calib_use_checkbox">
          <property name="text">
           <string>Use Calibration</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Rectangle [px]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QFormLayout" name="formLayout_7">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Top</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="calib_top_spinbox">
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="calib_width_spinbox">
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="calib_height_spinbox">
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="calib_left_spinbox">
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="1">
         <layout class="QFormLayout" name="formLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TaurusValueSpinBox" name="calib_actual_width_spinbox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TaurusValueSpinBox" name="calib_actual_height_spinbox"/>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Actual size [mm]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Limit image frame rate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="max_framerate_spinbox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusLabel</class>
   <extends>QLabel</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
  <customwidget>
   <class>TaurusValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>TaurusValueSpinBox</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>TaurusValueComboBox</class>
   <extends>QComboBox</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>TaurusValueCheckBox</class>
   <extends>QCheckBox</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>MAXLineEdit</class>
   <extends>TaurusValueLineEdit</extends>
   <header>maxwidgets.input</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
